.github/workflows/e2e_test_ci_models.yml

```diff
@@ -194,4 +202,73 @@ jobs:
           killall spice || true
           cat spice.log
 
-  
\ No newline at end of file
+  test_hf_model:
+    name: "huggingface model (${{ matrix.target.target_os }}-${{ matrix.target.target_arch }})"
```

Comment:
Quotes are not unified, use `"` or `'` instead of both

Reply:
Configured - than you